docs/user-guide.md

```diff
@@ -19,6 +19,20 @@ Below are simple steps to use this portal:
 
 {% embed url="https://www.youtube.com/playlist?list=PLJH-POb_55z8YYS_qAk_QNBQeiQ2VrtZD" %}
 
+## Compliance Toolkit Landing Page
+
+On the CTK landing page, partners can view the introduction of the CTK portal and watch a video about CTK. Additionally, they can login or register by clicking the respective buttons. The landing page also provides links to other resources such as the GitHub repository, documentation, and partner details.
+
+![CTK Landing Page1](\_images/ctk-landing-page1.png)
+
+Also, it highlights some of CTK's powerful features, like its pre-built test cases, validations, and detailed reporting.
+
+![CTK Landing Page2](\_images/ctk-landing-page2.png)
+
+It also covers the details of the SBI, Android SBI, SDK, and ABIS specs. By clicking the download option on the landing page, partners can also get the compliance toolkit apk.
+
+![CTK Landing Page3](\_images/ctk-landing-page3.png)
+
 ## Login
 
 The partners using the compliance tool kit should be registered in the system. Registered partners can log in to the compliance tool kit with the same credentials as they used to register themselves into the [Partner Management Portal](https://docs.mosip.io/1.2.0/modules/partner-management-services/partner-management-portal).
```

Comment:
Partners utilizing the compliance toolkit must be registered in the system. Once registered, partners can log in to the compliance toolkit using the same credentials they used for registration in the Partner Management Portal.